taca/utils/bioinfo_tab.py

```diff
@@ -60,12 +60,7 @@ def update_statusdb(run_dir):
     view = db.view('latest_data/sample_id')
     #Construction and sending of individual records, if samplesheet is incorrectly formatted the loop is skipped
     for flowcell in project_info:
-        if flowcell == 'UNKNOWN':
-            obj={'run_id':run_id, 'project_id':'ERROR_Samplesheet'}
-            logger.info("INVALID SAMPLESHEET, CHECK {} FORMED AT {}".format(run_id, valueskey))
-            error_emailer('no_samplesheet', run_id)
-            db.save(obj)
-        else:
+        if not flowcell == 'UNKNOWN':
```

Comment:
I noticed that in this code there is a lot of redundancy with things already implemented 

see this
https://github.com/SciLifeLab/TACA/blob/master/taca/illumina/Runs.py#L116
and 
function get_status